dev-db/etcd-wrapped/etcd-wrapped-0.4.9.ebuild

```diff
@@ -0,0 +1 @@
+files/etcd-wrapped.ebuild
```

Comment:
Lets not bother 0.4.9. that version is only still around because we don't have an established pattern for deprecating and removing things safely from the image yet. It isn't worth touching with any new development like this.